docs/configuration.md

```diff
@@ -1,4 +1,41 @@
-# Configuration
+page_title: Configure a Registry
+page_description: Explains how to deploy a registry service
+page_keywords: registry, service, images, repository
+
+
+# Configure a Registry
+
+
+## A simple development configuration
+
+The registry server can be configured with a yaml file. The following is a
+simple example that can used for local development:
+
+```yaml
+version: 0.1
+log: 
+	level: debug
+storage:
+    filesystem:
+        rootdirectory: /tmp/registry-dev
+http:
+    addr: localhost:5000
+    secret: asecretforlocaldevelopment
+    debug:
+        addr: localhost:5001
+```
+
+The above configures the registry instance to run on port 5000, binding to
+"localhost", with the debug server enabled. Registry data will be stored in
+"/tmp/registry-dev". Logging will be in "debug" mode, which is the most
+verbose.
+
+A similar simple configuration is available at [cmd/registry/config.yml],
+which is generally useful for local development.
+
+**TODO(stevvooe): Need a "best practice" configuration overview. Perhaps, we can point to a documentation section.
```

Comment:
This TODO belongs in either deploying or elsewhere. We can add advice and best practices here, but the full best practice config belongs in deploying.